ion_phys/tests/test_tls.py

```diff
@@ -0,0 +1,81 @@
+""" Simple rate equations example of a 2 level system"""
+
+import unittest
+
+from ion_phys.ions.ca43 import Ca43, ground_level, P32
+from ion_phys import Laser
+from ion_phys.rate_equations import Rates
+
+
+def steady_state(intensity):
+    "Steady state poulation in the P-state for resonant intensity /I0"
+    return intensity / (2 * intensity + 1)
+
+
+class TestTLS(unittest.TestCase):
+    """two-level system tests"""
+    def test_steady_state_intensity(self):
+        """Check that the steady state intensity scaling"""
+
+        # use both integers and floats
+        intensity_list = [1e-3, 1e-1, 0.3, 1, 1.0, 2, 10., 1.2e4]
+
+        ion = Ca43(B=5e-4, level_filter=[ground_level, P32])
```

Comment:
FWIW, I imagined defining a dummy Ion level structure with just two states for this example, but on second thoughts, this is probably nicer. Might be worth a comment that we're taking advantage of the cycling transition between the two stretched states.